CHANGELOG.md

```diff
@@ -1,8 +1,93 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Serialization Versioning](VERSIONING.md).
```

Comment:
I remove this because we're no longer following serialization versioning (shouldn't this be semantic?).